graphql/resolvers.go

```diff
@@ -1254,6 +1254,16 @@ func (r *queryResolver) DistroTaskQueue(ctx context.Context, distroID string) ([
 	return taskQueue, nil
 }
 
+func (r *taskQueueItemResolver) ExpectedDuration(ctx context.Context, obj *restModel.APITaskQueueItem) (string, error) {
+	expectedDuration := obj.ExpectedDuration.Round(time.Second).String()
+
+	if expectedDuration != "0s" {
```

Comment:
it's best if we include this `!= "0s"` condition in the format duration function since we also do this in other places we are formatting duration.